<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTransactionDialog</class>
 <widget class="QDialog" name="AddTransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="transactionDetails">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>166</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="expenseRadio">
       <property name="text">
        <string>Expense</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="incomeRadio">
       <property name="text">
        <string>Income</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>82</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="commentLineEdit"/>
     </item>
     <item row="2" column="3">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="categoriesComboBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="sumLineEdit"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sum</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTransactionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTransactionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
